spec/requests/update_endpoint_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe "PATCH /districts/:district/endpoints/:endpoint", type: :request do
```

Comment:
Can we also write a test for the legacy case where the endpoint district is nil?

Reply:
@davidsiaw I think `district` cannot be nil when creating an endpoint.
the original code is 
```
  def create
    endpoint = @district.endpoints.create!(permitted_params)
    render json: endpoint
  end
```